### PR TITLE
Removes a ton of unused code

### PR DESCRIPTION
**Pull request by Shadowlight213**
`<Shadowmobile>: Joan, said logic applies to every single adminspawn only thing ever`
`<Shadowmobile>: Should we go remove that?`
`<%Joan>: Yes`
`<%Joan>: Yes we should`
:cl:
del: Removed unused items
/:cl:

Probably part one of many as I'm certain I missed a few things.
I didn't include the mech teleporter in removal, as that can technically be made in RND if you get levels that high(which requires deconning a mech teleporter). Wasn't sure if I should remove the RND thing as well.

### CODE REVIEW

**Comment by jaxy15** [comment]
Syndrones are great, fuck off

**Comment by phil235** [comment]
Removing the Reticence??

**Review by Razharas — APPROVED** [review]
"X-only" items should die in hell tbh smh fams

**Comment by Shadowlight213** [comment]
@phil235 Thanks for reminding me. I guess I forgot about that one.

**Comment by ChangelingRain** [comment]
Don't forget the deathsquad.

**Comment by Incoming5643** [comment]
I know the point you're trying to make, but even risking that someone will yolomerge this is unacceptable.

**Comment by Shadowlight213** [comment]
@phil235 Yes. It's admin only.
@ChangelingRain I excluded DS because it was spawned via button, but sure. I can do an ERT removal as well.

**Comment by Shadowlight213** [comment]
@Incoming5643 I expect that it will get closed because I'm removing a maintainer's codebabby, despite said babby not being obtainable in game at all and never has been used since it was pr'd really.

**Comment by ChangelingRain** [comment]
It's a shame you can't remove ninja with this, because it is used...

**Comment by Shadowlight213** [comment]
@ChangelingRain Ikr?

**Comment by optimumtact** [comment]
Can you stop?

**Comment by Iamgoofball** [comment]
these weren't admin only originally

they got made admin only just to get away with removing them down the line specifically like this PR

**Comment by optimumtact** [comment]
no goof that's not even remotely true

File: code/modules/research/stock_parts.dm [hunk 1]
```diff
@@ -294,11 +294,4 @@
 	icon_state = "subspace_transmitter"
 	desc = "A large piece of equipment used to open a window into the subspace dimension."
 	origin_tech = "magnets=2;materials=2;bluespace=2"
-	materials = list(MAT_METAL=50)
-
-/obj/item/weapon/research//Makes testing much less of a pain -Sieve
```
Comment (Iamgoofball) [inline code/modules/research/stock_parts.dm]: this is literally a fucking debug testing item

are you retarded

**Comment by Iamgoofball** [comment]
@optimumtact oh wait nvm this PR's just fucking retarded

File: code/modules/research/stock_parts.dm (hunk 1 above)
Comment (Shadowlight213) [inline code/modules/research/stock_parts.dm]: @Iamgoofball see Joan's quote.

**Pull request closed without merging**